### DISCUSSION TITLE
Undesirable dark mode icon behavior

### DISCUSSION THREAD

**Discussion by JamesHenry** [johnlindquist/kit#788] [Errors] [answered]
Hi John!

I just updated to the latest (was way behind on 1.17.x before), and now my icon is behaving differently.

Now when in dark mode (on my external monitor) the icon is now always white, despite the fact that it should be black based on the background it is on, as all my other menu bar items are. It was always appropriately black on 1.17.x

See video where I toggle dark mode (it is white when in dark mode, black when in light mode) back and forth - the Script Kit icon is the only one that changes. All the others are appropriately colored based on the background:

https://user-images.githubusercontent.com/900523/175782826-6f11eecc-3af7-4cc3-92a0-c501587fa81a.mov

**Comment by johnlindquist** [comment]
@JamesHenry Couple questions:

1. Are you on 1.19.11?
2. Which OSX version are you on? 
    OSX treats dark mode differently before/after Big Sur and if users have enabled "reduce transparency" in their accessibility settings.  I've testing on Catalina and Monterey, I may be missing something in another OSX version...

## Why?
Since I want to use status colors (green, orange, red, etc) in the icon, I have to manually adjust the icon rather than just providing a default "template" icon that OSX would adjust for me.

## Discord
We're starting up a Discord server if you want to join/chat it out there: https://discord.gg/BR3C7R9b

**Reply by JamesHenry** [reply]
Thanks @johnlindquist I've joined the discord now.

Yes 1.19.11, it's macOS 12.4 (Monterey).

I have realized the external monitor bit is significant:

I tried restarting my laptop from scratch and disconnected everything. Script Kit icon was black on the purplish default menu bar coloring (along with all other icons).

As soon as I plug my external monitor in, the menu bar goes light (regardless of if the laptop lid is open or closed) and the Script Kit icon changes (no other icons do).

Then even if I disconnect the external monitor again, the menu bar stays light, and there seems to be no way short of a restart of the machine to restore it...

**Comment by johnlindquist** [comment]
Dang, I just reproduced on Big Sur. I'll get it fixed ASAP

**Comment by JamesHenry** [comment, marked as answer]
Resolved in 1.19.2! Thanks @johnlindquist
